readme.txt

```diff
@@ -90,6 +90,7 @@ Gateways:
 * cpsms.dk
 * bulksmshyderabad.co.in
 * ozioma.net
+* Bulutfon.com
```

Comment:
Dear @mostafasoufi,

First off all thank you very much for this great plugin.

I'm the co-founder of Bulutfon. If it isn't problem for you, you can also add Bulutfon to premium list. We'll send email to our clients that `wp-sms` plugin and `wp sms professional package` are both integreted with Bulutfon. If they need some more feature like Buddy press integration, they may choice `wp sms professional package` . 

Regards.